content/en/agent/guide/how-do-i-uninstall-the-agent.md

```diff
@@ -12,8 +12,9 @@ further_reading:
 Choose your platform to see dedicated instructions to uninstall the Agent:
 
 ## Debian and Ubuntu
-**Agent v6 & v7**
 
+{{< tabs >}}
+{{% tab "Agent v6 & v7" %}}
```

Comment:
I know this is common in our docs, but it's not recommended from an accessibility perspective:
https://readabilityguidelines.co.uk/grammar-points/ampersands/

Feel free to disregard this suggestion if you think it's more important to maintain uniformity. In case it's helpful, I took a screenshot of my local build with this change, and I think it looks fine:
https://a.cl.ly/E0uZG9pp

Reply:
No worries, good call